Lib/test/test_syntax.py

```diff
@@ -465,7 +465,7 @@
 # SyntaxError: expression cannot contain assignment, perhaps you meant "=="?
 # >>> f(True=2)
 # Traceback (most recent call last):
-# SyntaxError: cannot assign to True
+# SyntaxError: cannot assign to True. Maybe you meant '==' instead of '='?
```

Comment:
The suggestion is questionable here. Wouldn't a more likely cause be that the user just misspelled a keyword arg and accidentally used a reserved word? Or that they want to use a reserved word as a keyword arg (e.g. in a `**kwargs` case)? Passing a Boolean expression of the form `foo == bar` as an argument must be pretty rare, and mistyping it with `=` instead of `==` even more so.

Reply:
Hummmm, good point. I can try to see if I can exclude the suggestion inside function arguments but I am not sure how easy it could be. Note that for keywords it won't happen:

```
ne 1
    f(if =3)
      ^
SyntaxError: invalid syntax
```

Reply:
We could suppress this for the common singletons like `True`, `False`, `None`...